packages/components/pie-webc-core/src/decorators/test/required-property.spec.ts

```diff
@@ -0,0 +1,98 @@
+import {
+    beforeEach,
+    afterEach,
+    describe,
+    it,
+    expect,
+    vi,
+} from 'vitest';
+
+import { requiredProperty } from '../required-property';
+
+describe('requiredProperty', () => {
+    let consoleErrorSpy: unknown;
+
+    beforeEach(() => {
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        consoleErrorSpy = vi.spyOn(console, 'error').mockImplementation(() => {});
+    });
+
+    afterEach(() => {
+        vi.restoreAllMocks();
+    });
+
+    // Mock class to test the decorator
+    class MockComponent {
+        @requiredProperty('mock-component')
+            color!: string;
```

Comment:
Will changing this to `color?` mean that the ts comments below can be removed?

Reply:
It does! Thank you!